src/backend/pluginLoaders.js

```diff
@@ -97,8 +105,25 @@ export async function updateMirror(
       github.updateMirror(project.repoIds, githubToken, cache, reporter)
     );
   }
+
+  let loadedInitiativesDirectory;
```

Comment:
when creating undefined variables like this, we should specify the type.

Reply:
Done